crates/router/src/configs/defaults.rs

```diff
@@ -742,7 +760,7 @@ impl Default for super::settings::RequiredFields {
                                             (
                                                 "billing.address.first_name".to_string(),
                                                 RequiredFieldInfo {
-                                                    required_field: "billing.address.first_name".to_string(),
+                                                    required_field: "payment_method_data.billing.address.first_name".to_string(),
                                                     display_name: "billing_first_name".to_string(),
                                                     field_type: enums::FieldType::UserBillingName,
```

Comment:
this should be UserFullName

Reply:
Should I also change this to `card_holder_name`?